spec/support/service_spec_helper.rb

```diff
@@ -28,7 +28,7 @@ def topological_url(partial_path, api_version = "v2.0")
     url.to_s
   end
 
-  def approval_url(partial_path, api_version = "v1.0")
+  def approval_url(partial_path, api_version = "v1.2")
```

Comment:
Since this looks like something we have to change often it would be better to have it as a constant at the top of the file instead of hidden here.  Hopefully we can figure out a better solution, but does not have to be in this PR.

Reply:
@gmcculloug I have updated to use a constant.